opengemini-client-api/src/main/java/io/opengemini/client/api/RetentionPolicy.java

```diff
@@ -0,0 +1,24 @@
+package io.opengemini.client.api;
+
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+public class RetentionPolicy {
+    String name;
+
+    String duration;
+
+    String shardGroupDuration;
+
+    String hotDuration;
+
+    String warmDuration;
+
+    String indexDuration;
+
+    int replicaNum;
+
+    Boolean isDefault;
```

Comment:
can we use boolean instead of boxed type?

Reply:
ok